test/docker/networktest/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# escape=`
+FROM microsoft/windowsservercore
+
+SHELL ["powershell.exe","-command"]
+
+# the source msi should change on a daily basis
+# the destination should not change
+ADD PSCore.msi /PSCore.msi
```

Comment:
We don't have the whole package in MyGet, but I've given Jim feedback on how to get the latest MSI from AppVeyor.